README.md

```diff
@@ -1,3 +1,7 @@
+> [!NOTE] 
```

Comment:
Uses GitHub specific markdown syntax, will not work elsewhere
```suggestion
> **Note**
```

![image](https://github.com/user-attachments/assets/88ebb0c9-455c-4406-a914-5cc07f351673)